sklearn/multioutput.py

```diff
@@ -321,6 +321,19 @@ class MultiOutputClassifier(ClassifierMixin, MultiOutputEstimator):
     ----------
     estimators_ : list of ``n_output`` estimators
         Estimators used for predictions.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.datasets import make_classification
+    >>> from sklearn.multioutput import MultiOutputClassifier
+    >>> from sklearn.neighbors import KNeighborsClassifier
+
+    >>> X, y = make_classification(n_features=4, random_state=0)
+    >>> y_multi = np.array([y, y]).T
+    >>> clf = MultiOutputClassifier(KNeighborsClassifier()).fit(X, y_multi)
+    >>> clf.predict(np.array([0.2, 0.5, -0.4, 0.3]).reshape(1, -1))
```

Comment:
is the `reshape` really needed?

Reply:
Thanks, new commit with simplification added.